src/client/inputhandler.h

```diff
@@ -170,7 +172,9 @@ class MyEventReceiver : public IEventReceiver
 
 		mouse_wheel = 0;
 	}
-
+	void setScript(ClientScripting* script){
```

Comment:
The input handler having direct access to the script API feels like a code smell to me. Would be better to have some callback and hide it from the input classes

Reply:
Also wrong pointer and brace placement. http://dev.minetest.net/Code_style_guidelines
```C++
void setScript(ClientScripting *script)
{
	m_script = script;
}
```